src/main/java/com/theelixrlabs/healthcare/constants/DoctorPatientAssignmentConstants.java

```diff
@@ -8,6 +8,7 @@ public class DoctorPatientAssignmentConstants {
     public static final String DOCTOR_ALREADY_ASSIGNED_KEY = "doctor.already.assigned";
     public static final String DOCTOR_ALREADY_UNASSIGNED_KEY = "doctor.already.unassigned";
     public static final String DOCTOR_ID_INVALID_KEY = "doctor.id.invalid";
+    public static final String DOCTOR_ID_PARAM = "doctorId";
     public static final String DOCTOR_NOT_FOUND_KEY = "doctor.id.not.found";
     public static final String NO_ASSIGNMENT_EXISTS_KEY = "assignment.not.exists";
     public static final String PATIENT_ID_INVALID_KEY = "patient.id.invalid";
```

Comment:
Rename PATIENT_ID_INVALID_KEY to INVALID_PATIENT_ID_KEY

Reply:
Renamed PATIENT_ID_INVALID_KEY to INVALID_PATIENT_ID_KEY.